sympy/solvers/solvers.py

```diff
@@ -3144,7 +3144,7 @@ def _invert(eq, *symbols, **kwargs):
 
     >>> invert(sqrt(x + y) - 2)
     (4, x + y)
-    >>> invert(sqrt(x + y) - 2)
+    >>> invert(sqrt(x + y) + 2)
     (4, x + y)
```

Comment:
Why is this changed?

Reply:
it was a typo; the line above is identical